tas-initializer/Dockerfile.tas-initializer

```diff
@@ -0,0 +1,25 @@
+FROM registry.redhat.io/rhel9/go-toolset@sha256:f969e6a4fe53663f3fc8d0ddc17ec2fc3def2af8e68ce7061f016a598798f474 AS build-env
+
+WORKDIR /opt/app-root/src
+
+RUN curl -sL https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/stable/openshift-client-linux.tar.gz -o /opt/app-root/src/openshift-client-linux.tar.gz && \
+    tar xvf /opt/app-root/src/openshift-client-linux.tar.gz
+
+FROM registry.access.redhat.com/ubi9/ubi@sha256:6b95efc134c2af3d45472c0a2f88e6085433df058cc210abb2bb061ac4d74359
```

Comment:
Is this needed?

Reply:
Can check, the issue was curl with the other image, wouldnt run